<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>serialhandler</class>
 <widget class="QDialog" name="serialhandler">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>623</width>
    <height>215</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_3">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item row="0" column="0">
    <widget class="QWidget" name="gridWidget" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <layout class="QGridLayout" name="gridLayout_4">
      <item row="0" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QGroupBox" name="selectBox">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Ignored" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>150</width>
            <height>0</height>
           </size>
          </property>
          <property name="baseSize">
           <size>
            <width>0</width>
            <height>0</height>
           </size>
          </property>
          <property name="title">
           <string>Escolha a porta serial:</string>
          </property>
          <layout class="QGridLayout" name="gridLayout">
           <item row="6" column="0">
            <widget class="QLabel" name="pidLabel">
             <property name="text">
              <string>Produto ID:</string>
             </property>
            </widget>
           </item>
           <item row="3" column="0">
            <widget class="QLabel" name="serialNumberLabel">
             <property name="text">
              <string>Número de série:</string>
             </property>
            </widget>
           </item>
           <item row="4" column="0">
            <widget class="QLabel" name="locationLabel">
             <property name="text">
              <string>Porta:</string>
             </property>
            </widget>
           </item>
           <item row="5" column="0">
            <widget class="QLabel" name="vidLabel">
             <property name="text">
              <string>Vendedor ID:</string>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="descriptionLabel">
             <property name="text">
              <string>Descrição:</string>
             </property>
            </widget>
           </item>
           <item row="2" column="0">
            <widget class="QLabel" name="manufacturerLabel">
             <property name="text">
              <string>Fabricante:</string>
             </property>
            </widget>
           </item>
           <item row="0" column="0">
            <widget class="QComboBox" name="serialPortInfoListBox"/>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="parametersBox">
          <property name="title">
           <string>Parametros porta serial</string>
          </property>
          <layout class="QGridLayout" name="gridLayout_2">
           <item row="0" column="1">
            <widget class="QComboBox" name="baudRateBox"/>
           </item>
           <item row="0" column="0">
            <widget class="QLabel" name="baudRateLabel">
             <property name="text">
              <string>BaudRate:</string>
             </property>
            </widget>
           </item>
           <item row="2" column="0">
            <widget class="QLabel" name="label_2">
             <property name="text">
              <string>Parity:</string>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="label">
             <property name="text">
              <string>Data bits:</string>
             </property>
            </widget>
           </item>
           <item row="3" column="0">
            <widget class="QLabel" name="label_3">
             <property name="text">
              <string>Stop bits</string>
             </property>
            </widget>
           </item>
           <item row="4" column="0">
            <widget class="QLabel" name="label_4">
             <property name="text">
              <string>Flow Control:</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QComboBox" name="dataBitsBox"/>
           </item>
           <item row="2" column="1">
            <widget class="QComboBox" name="parityBox"/>
           </item>
           <item row="3" column="1">
            <widget class="QComboBox" name="stopBitsBox"/>
           </item>
           <item row="4" column="1">
            <widget class="QComboBox" name="flowControlBox"/>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
